Sources/OutputStream+Write.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  OutputStream+Write.swift
+//  FTAPIKit
+//
+//  Created by Matěj Kašpar Jirásek on 27/02/2019.
+//  Copyright © 2019 FUNTASTY Digital s.r.o. All rights reserved.
+//
+
+import Foundation
+
+extension OutputStream {
+    private static let streamBufferSize = 1024
+
+    func write(inputStream: InputStream) throws {
+        inputStream.open()
+        defer { inputStream.close() }
+
+        while inputStream.hasBytesAvailable {
```

Comment:
[Speculation:] I have no time left so I’ll kep it short. I wonder, whether this code would slow an app significantly if run on main queue. All implementation I found were working with `stream(_:handle:)` delegate that is called when stream is scheduled on a run loop. I wonder there must be two cases: 1) All writes and reads block until they can write or read or -> major performance hit when run on Main queue 2) Writes and reads doesn’t wait for stream to create space -> write/read could fail.  🤔

Reply:
I think it is outside of scope of this PR, when we encounter some performance issues it would be the right time to investigate this. Premature omtimization is not always a good idea.